src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -44,9 +44,10 @@
  * </p>
  * <p>
  * For additional restriction of type usage see also:
- * <a href="https://checkstyle.org/checks/coding/illegalinstantiation.html">
+ * <a href="https://checkstyle.org/checks/coding/illegalinstantiation.html#IllegalInstantiation">
```

Comment:
Why do we need anchors here still?

Reply:
There were two solutions here, either remove them from xdoc or keep them in javadoc. I remember we talked about keeping them to make it easy for mobile.

Reply:
Ok, I am leaving this unresolved for next reviewer to see, but I am good.

Reply:
We will meditate on this at the end https://github.com/checkstyle/checkstyle/issues/13386